.github/workflows/release.yml

```diff
@@ -61,8 +61,9 @@ jobs:
       env:
         crate: ${{ needs.info.outputs.crate }}
         CARGO_REGISTRY_TOKEN: ${{ secrets.CARGO_REGISTRY_TOKEN }}
-      run: cargo publish -p "$crate"
+      run: cargo publish -p "$crate" --allow-dirty
```

Comment:
Can we also upload the public key as a release artifacts so that users and taiki-e/install-action can download and use that?